Utils/hdinsight-node-common/pom.xml

```diff
@@ -187,5 +239,14 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jre8</artifactId>
```

Comment:
I suppose the Kotlin lang support is not only for test code, but also for the whole project